spinnaker-config/powerdummy-populator/powerdummy.yaml

```diff
@@ -0,0 +1,21 @@
+schema: "1"
+id: powerdummyPopulator
+pipeline:
+  application: powerdummy
+  name: Repopulate powerdummy tenant
+  template:
+    source: spinnaker://deploySingleClusterFromPubsub
+  variables:
+    githubRepository: powerdummy-populator
+    kubernetesAccount: cognitedata-development
+    kubernetesManifests: manifests.yaml
+    dockerOrganization: cognitedata-development
```

Comment:
delete this line. The default org is cognitedata, and the images are pushed there from jenkins.

Reply:
fixed